src/types/types.d.ts

```diff
@@ -19,6 +19,12 @@ export interface Keybind {
 
 export type ActionOrKeybind = Action | Keybind;
 
+export interface OpenSpaceVersion {
+  latestOpenSpaceVersion?: { major: number; minor: number; patch: number };
```

Comment:
See previous comment about making a type for the version number

Reply:
Also, this type (with the three different version states) should be moved to the versionSlice.ts file?

Reply:
Hehe so Ylva said the complete opposite. But we just had a new discussion about it and concluded that we should always have an interface with the name `xyzState` to describe the `initialState` of the redux store and any other interfaces or types should be in the @types file. So in this case I'll make a `export type OpenSpaceVersionState = OpenSpaceVersion` in the versionSlice.ts file and keep the OpenSpaceVersion in the types file

Reply:
What I meant is NOT that the `{ major: number; minor: number; patch: number };` interface should be moved to the slice, but this one. 

```
{
  latestOpenSpaceVersion?: { major: number; minor: number; patch: number };
  openSpaceVersion?: { major: number; minor: number; patch: number };
  socketApiVersion?: { major: number; minor: number; patch: number };
 }
 
 I don't see a reason why this should be anywhere else than in the slice where it is used

Reply:
Also, how about. 
```
interface SemanticVersion {
  major: number;
  minor: number;
  patch: number
}
```
?